### PR TITLE
Add DevTools meet pic

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

**Review by diogogmatos — APPROVED** [review]
🚀

**Comment by feliciofilipe** [comment]
I don't this one is worthy of being here 😅

**Comment by MarioRodrigues10** [comment]
I think it deserves a space in our world map because it was a tech event and you were with a CeSIUM tshirt

File: data/places.ts [hunk 1]
```diff
@@ -322,5 +322,21 @@ export const PLACES: IPin[] = [
     coordinates: [48.85946363288889, 2.287131396878628],
     date: '2022-07-17',
     photo: '/photos/matildeopbravo/paris_camoes.jpg'
+  },
+  {
+    author: [
+      'Filipe Felício',
+      'Nélson Estevão',
```
Comment (nelsonmestevao) [inline data/places.ts:329]:
```suggestion
      'Nelson Estevão',
```

**Pull request closed without merging**